packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -0,0 +1,69 @@
+import { IntlUtils } from '@common/intl';
+import {
+  isValid,
+  differenceInMonths,
+  isPast,
+  isThisWeek,
+  isToday,
+  isThisMonth,
+  isAfter,
+  isBefore,
+  isEqual,
+  format,
+  parseISO,
+  fromUnixTime,
+} from 'date-fns';
+import { enGB, enUS, fr, ar } from 'date-fns/locale';
```

Comment:
Not thrilled about having to import these. Ideally we would lazy-load them as required, as part of the Intl Context (updating when "changeLanguage" is run), but not sure how to do this / if it's worth it.

Reply:
agree - not ideal. would be nice to lazy load if we can.. but that's a problem for another day!